### ISSUE TITLE
Expanded Reference Photo

### ISSUE THREAD

**Issue by Ashotep** [Barleytree/Dupery#6]
So, I made myself a updated reference guide photo that includes your Cassidy and the Styles from So Many Dupes.  I thought you may like to include it as an optional reference if you like.  So, I'm uploading the .png photo and the .xcf file.

This took me longer then I want to admit because I basically had to teach myself how to use Gimp.  I don't mind.  Feel free to discard or add if you want.  Or take what I have done and change it up a bit.

[Google Drive](https://drive.google.com/drive/folders/1YNUPrFbOMre0U_7rjw95dZy6gwHeiq9g?usp=sharing)

**Comment by Barleytree** [comment]
Gosh, that's a great reference image, thank you for taking the time to do that! I will gladly use it in the guide in place of the inferior one I made (creditting you, of course).

**Comment by Ashotep** [comment]
Yeah, I took your original as inspiration because it was so easy to read.  I basically wanted the exact same thing but expanded.

**Comment by Ashotep** [comment]
Updated the image to make it accurate.  Sorry for the text wall in the comment section on steam.  I also found and added the hermits hair and added.  There are more missing eyes, bodies and faces that I found while playing around with unknown numbers.  This took a bit because if you use and invalid number the game crashes.  So it was a process of making an edit, booting up the game and seeing if something worked or if it would crash the game.  rinse repeat.

[Updated photo](https://photos.app.goo.gl/TMqykPCsDdVfig6H6)
